ansible_collections/arista/avd/roles/eos_designs/docs/tables/platform-families.md

```diff
@@ -0,0 +1,621 @@
+            vni_decap: <bool; default=True>
+            vni_encap: <bool; default=True>
+            vtep_decap: <bool; default=True>
+            vtep_encap: <bool; default=True>
+
+          # Support for configurable speeds per speed-group.
+          # The feature will be ignored on platforms where this is false.
+          hardware_speed_group: <bool; default=True>
+
+          # Support for PVLAN feature.
+          # On platforms with additional requirements to use PVLAN, ex. 7280R/R2/R3 set this via "private_vlan" in the specific platform settings.
+          # See the TOI at https://www.arista.com/en/support/toi/eos-4-25-0f/14609-support-for-private-vlan.
+          private_vlan: <bool; default=True>
+
+          # Support for sFlow.
+          # The feature will be ignored on platforms where this is false.
+          sflow: <bool; default=True>
+
+          # Support for sFlow on sub-interfaces.
+          # The feature will be ignored on platforms where this is false.
+          sflow_subinterfaces: <bool; default=True>
+
+          # Support for Arista WAN features.
+          # An error will be raised if the feature is enabled and this is false.
+          wan: <bool; default=True>
+
+          # Support for Precision Time Protocol (PTP).
+          # The feature will be ignored on platforms where this is false.
+          ptp: <bool; default=True>
+
+        # Entropy source improves the randomness of the numbers used to generate MACsec's cryptographic keys.
+        security_entropy_sources:
+
+          # Use a hardware based source.
+          hardware: <bool>
+
+          # Use the HAVEGE algorithm.
+          haveged: <bool>
+
+          # Use the Jitter RNG algorithm of a CPU based source.
+          cpu_jitter: <bool>
+
+          # Only use entropy from the hardware source.
+          hardware_exclusive: <bool>
+
+        # PREVIEW: This option is marked as "preview", meaning the data models or generated configuration can change at any time.
+        # Digital Twin settings applied when `avd_digital_twin_mode` is `true`.
+        digital_twin:
+
+          # Name of an alternate `platform_settings` platform used when running in Digital Twin mode.
+          # The `platform_settings` for the regular `platform` is used if this is not set.
+          platform: <str>
+
+          # ACT node type.
+          act_node_type: <str; "cloudeos" | "cvp" | "generic" | "third-party" | "tools-server" | "veos">
+
+        # Settings for hardware validation performed by the `anta_runner` role.
+        # If `enabled` is set to `false`, all other keys in this dictionary are ignored.
+        #
+        # For the `min_*` keys:
+        # - Undefined (Default): Validate that all available slots are populated.
+        # - Positive Integer: Validate that the number of components inserted is at least the specified minimum.
+        # - 0: Skip the validation for this specific component.
+        validate_hardware:
```

Comment:
We should consider if this should move to the sku.

Reply:
`validate_hardware` should be under node_type/device_profiles. This has not been released, so we can change it easily.

For example, the same SKU may be used in different roles, spine vs. leafs, with varying expectations of redundancy for power supplies, etc.

Reply:
@carl-baillargeon any thoughts here?